### ISSUE TITLE
Any plans for librabbitmq-2.0.0 release?

### ISSUE THREAD

**Issue by mrueg** [celery/librabbitmq#81]
Hi, 

I see the last version was released about 1 year ago. Therfore, I'd like to know if there are any plans to release another versions in the 1.6 branch?

Thanks!

**Comment by mrueg** [comment]
@ask friendly ping? :)

**Comment by cmpis** [comment]
Same question

**Comment by cmpis** [comment]
@mrueg I figure the reason, after looking at the issues/PRs more, is that they want to work to get Py3 support (#13) but that is probably a lot of work in the C code to make it compatible with Python 3 and stay compatible with Python 2.

**Comment by cclauss** [comment]
Folks...  Would it be possible to get a new release?  There are already fixes in this repo that should be delivered in a release.  https://github.com/fossasia/query-server/issues/123#issuecomment-321842390

If development is halted, is there an alternative library that you would recommend?

**Comment by auvipy** [comment]
pyamqp is now recommended

**Comment by auvipy** [comment]
https://github.com/celery/librabbitmq/releases/tag/v2.0.0 should be soon on pypi with py3 support

**Comment by auvipy** [comment]
@thedrow can you push this to pypi?

**Comment by thedrow** [comment]
We need to update the release notes.

**Comment by nikodemkler** [comment]
@thedrow is there any progress with updating release notes and as a result pushing v2.0.0 to PyPI?

**Comment by thedrow** [comment]
Nope, sorry. Too busy right now. We'll welcome an update to the release notes.

**Comment by matusvalo** [comment]
Hi all, I will try to find some time and write down release notes. I hope I will do it by the end of week.

**Comment by thedrow** [comment]
Released.
